adr/context.py

```diff
@@ -192,23 +194,6 @@ def extract_context_names(query):
     return contexts
 
 
-def sort_context_dict(context_dict):
```

Comment:
Did you mean to remove this? Or was it a merge error?

Reply:
Yes I want to remove it, because it is unused. The OrderedDict is passed from function to function, so no need to have a sort function.